Gemfile.lock

```diff
@@ -2,67 +2,72 @@ PATH
   remote: .
   specs:
     fmadata_parse_name (1.0.0)
```

Comment:
You need to re-add Gemfile.lock to repo after you updated version number in version.rb
This should  update to `fmadata_parse_name (2.0.0)`